core/ui/ControlPanel/Modals/AddPlugins.tid

```diff
@@ -87,7 +87,14 @@ Search: <$edit-text tiddler="""$:/temp/RemoteAssetSearch/$(currentTiddler)$""" d
 \define display-server-connection()
 <$list filter="[all[tiddlers+shadows]tag[$:/tags/ServerConnection]suffix{!!url}]" variable="connectionTiddler" emptyMessage=<<load-plugin-library-button>>>
 
+<$reveal type='match' state=<<qualify $(currentTiddler)$>> text='show'>
```

Comment:
This use of the `qualify` macro ends up with a tiddler with the current tiddler title, a dash and a numeric prefix. In this case that will at least result in a system tiddler, but it won't be in the `$:/state` namespace that ensures that it is discarded during saving/syncing.

Better would be:

```
<$reveal type='match' state=<<qualify """$:/state/addPlugin/$(currentTiddler)$""">> text='show'>
```